aldryn_people/admin.py

```diff
@@ -13,10 +21,24 @@
 
 class PersonAdmin(AllTranslationsMixin, TranslatableAdmin):
     list_display = [
-        '__str__', 'email', 'vcard_enabled', ]
+        '__str__', 'email', 'vcard_enabled', 'num_groups', ]
     list_filter = ['groups', 'vcard_enabled']
     search_fields = ('translations__name', 'email', 'translations__function')
-    raw_id_fields = ('user',)
+
+    def __init__(self, *args, **kwargs):
+        """
+        Determines if the User widget should be a drop-down or a raw ID field.
+        """
+        # This is a hack to use until get_raw_id_fields() lands in Django:
+        # https://code.djangoproject.com/ticket/17881.
+        user_threshold = getattr(
+            'settings', 'ALDRYN_PEOPLE_USER_THRESHOLD', 50)
+        super(PersonAdmin, self).__init__(*args, **kwargs)
+        user_model = getattr(
+            settings, 'AUTH_USER_MODEL', 'auth.User').split('.')
+        model = get_model(user_model[0], user_model[1])
+        if model.objects.count() > user_threshold:
+            self.raw_id_fields = ('user', )
```

Comment:
admin instances are very dangerous when setting attributes like this, can cause odd behavior because they're not instantiated on every request, is there a get_raw_id_fields()?